### ISSUE TITLE
Crash report on 7.0: ISE in MediaUtils.downloadExternalMedia

### ISSUE THREAD

**Issue by hypest** [wordpress-mobile/WordPress-Android#5670]
```
Fatal Exception: java.lang.RuntimeException: An error occurred while executing doInBackground()
       at android.os.AsyncTask$3.done(AsyncTask.java:318)
       at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
       at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
       at java.util.concurrent.FutureTask.run(FutureTask.java:242)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
       at java.lang.Thread.run(Thread.java:761)
Caused by java.lang.IllegalStateException: Can't compress a recycled bitmap
       at android.os.Parcel.readException(Parcel.java:1691)
       at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:183)
       at android.database.DatabaseUtils.readExceptionWithFileNotFoundExceptionFromParcel(DatabaseUtils.java:146)
       at android.content.ContentProviderProxy.openTypedAssetFile(ContentProviderNative.java:692)
       at android.content.ContentResolver.openTypedAssetFileDescriptor(ContentResolver.java:1145)
       at android.content.ContentResolver.openAssetFileDescriptor(ContentResolver.java:982)
       at android.content.ContentResolver.openInputStream(ContentResolver.java:702)
       at org.wordpress.android.util.MediaUtils.downloadExternalMedia(MediaUtils.java:223)
       at org.wordpress.android.ui.media.MediaBrowserActivity$6.doInBackground(MediaBrowserActivity.java:778)
       at org.wordpress.android.ui.media.MediaBrowserActivity$6.doInBackground(MediaBrowserActivity.java:774)
       at android.os.AsyncTask$2.call(AsyncTask.java:304)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
       at java.lang.Thread.run(Thread.java:761)
```

**Comment by hypest** [comment]
A similar error was fixed in 6.9.1 with https://github.com/wordpress-mobile/WordPress-Android/issues/5333

**Comment by daniloercoli** [comment]
Closing this for now, since we've heavily changed the way external pictures are shared to wp-android.

I bet this issue was due to the old code that postponed the downloading of the shared picture in a background thread, making it unusable (already recycled) when required. See https://github.com/wordpress-mobile/WordPress-Android/issues/5818
